src/main/resources/xlr_xldeploy/XLDVersionsTile.py

```diff
@@ -12,26 +12,29 @@
 from xlr_xldeploy.XLDeployClientUtil import XLDeployClientUtil
 
 import org.joda.time.DateTime
+import org.joda.time.format.DateTimeFormat
 
 if not xldeployServer:
     raise Exception("XL Deploy server ID must be provided")
 
-today = org.joda.time.DateTime()
+#set the acceptable range of values for numberOfDays
 day_range = range(1,91)
```

Comment:
Shouldn't this 90 or 91 be a hidden property in the `synthetic.xml`?
So containing: `hidden="true"`

Reply:
I can do that, was confused as to if it just not throw error on the range or if you wanted it hidden.  I can update to have hidden="true"